docs/source/guides/configs/k8s.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/sh
+
+# the purpose of this scripts:
+# - setup minikube, kubectl, helm if doesn't exists through binary
+# - Run through most of configurations setup for k8s cluster.
+#
+# NOTES:
+# - This scripts should run as-is, meaning this script will run within BentoML documents scope.
+# - Example BentoService: aarnphm/bentoml-sentiment-analysis:latest
```

Comment:
maybe we should release an example service to docker hub and use it in our deployment related examples and docs?

Reply:
That would be best to do